server/lib/nl/detection/llm_fallback.py

```diff
@@ -122,6 +126,10 @@ def _has_no_entity(d: Detection) -> bool:
   return not d.places_detected or not d.places_detected.entities_found
 
 
+def _has_entity(d: Detection) -> bool:
```

Comment:
nit: I don't think we need this helper since there's not really any logic here

Reply:
The double negative is a little rough for readability -- I updated the code to only have _has_entity to avoid this issue. Is that okay?

Reply:
I'll merge this for now so that the logic is in autopush. I can change the readability part if you prefer the _has_no_entity with the double negative as a follow up!